ui/package.json

```diff
@@ -60,7 +60,6 @@
     "ember-cli-clipboard": "^0.8.0",
     "ember-cli-content-security-policy": "^1.0.0",
     "ember-cli-dependency-checker": "^3.0.0",
-    "ember-cli-eslint": "^4.2.3",
```

Comment:
I think we still want this as this will fail the build if something doesn't pass linting. Do you know if this still works when eslint is running prettier?

Reply:
Good catch! No, I don't think it will fail, so I can add this back in.